styles/styles.css

```diff
@@ -757,6 +757,16 @@ main .section.static > .tab-item.active > .section-container {
   margin-bottom: 5rem;
 }
 
+/* section - text align left */
+main .section.left-align-text h1,
```

Comment:
if its just for headings, maybe you can rename this to `left-align-headings`.  Also this may be useful beyond sections so you can add it as a pure mixin without `.section` in the selector.

Reply:
on second thoughts maybe `left-align-text` is ok. Though you can still consider making this a mixin.